rclcpp/src/rclcpp/node.cpp

```diff
@@ -138,7 +138,8 @@ Node::Node(
       node_services_,
       node_logging_,
       node_clock_,
-      node_parameters_
+      node_parameters_,
+      options.clock_qos(),
```

Comment:
Is the `,` going to work?

Reply:
Sorry, I compiled and tested the wrong workspace :man_facepalming:

Reply:
fixed